components/FeatureHighlight/src/MDCFeatureHighlightViewController.m

```diff
@@ -29,12 +26,14 @@ @interface MDCFeatureHighlightViewController () <UIViewControllerTransitioningDe
 @implementation MDCFeatureHighlightViewController {
   MDCFeatureHighlightAnimationController *_animationController;
   MDCFeatureHighlightCompletion _completion;
-  MDCFeatureHighlightView *_featureHighlightView;
   NSTimer *_pulseTimer;
   UIView *_displayedView;
   UIView *_highlightedView;
 }
 
+@synthesize outerHighlightColor = _outerHighlightColor;
+@synthesize innerHighlightColor = _innerHighlightColor;
```

Comment:
I don't think you should duplicate the color state in the view controller. Just make the getters return view.highlightColor.